script.js

```diff
@@ -1,12 +1,169 @@
 //You can edit ALL of the code here
+
+let allEpisodes = getAllEpisodes();
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+// headerContainer.classList.add("header-container");
+//Level 350 - Switch to fetching live data!
+function fetchAllEpisodes(series) {
+  return fetch(`https://api.tvmaze.com/shows/${series}/episodes`)
+    .then((response) => response.json())
+    .then((data) => {
+      allEpisodes = data;
+      makePageForEpisodes(allEpisodes);
+    })
+    .catch(function (error) {
+      console.log(error);
+    });
+}
+
+function tvSeriesSetup(showList) {
+  let selectSeries = document.getElementById("show-select");
+  let optionSeries = document.createElement("option");
+  optionSeries.textContent = "Please Choose Your Show";
```

Comment:
This option is not disabled, so it's still selectable. When I select default option after selecting a show, I get the following errors: 

```
Access to fetch at 'https://api.tvmaze.com/shows/Please%20Choose%20Your%20Show/episodes' from origin 'http://127.0.0.1:5500' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
script.js:11     GET https://api.tvmaze.com/shows/Please%20Choose%20Your%20Show/episodes net::ERR_FAILED 404
fetchAllEpisodes @ script.js:11
(anonymous) @ script.js:41
script.js:18 TypeError: Failed to fetch
    at fetchAllEpisodes (script.js:11:10)
    at HTMLSelectElement.<anonymous> (script.js:41:5)
```

It would be good to implement a solution for this case by either: 
1. Making this option disable
2. When this option is selected, return to initial state of the app